core/services/ardupilot_manager/ArduPilotManager.py

```diff
@@ -16,6 +16,8 @@
 from settings import Settings
 from Singleton import Singleton
 
+# pylint: disable=consider-using-with
+
```

Comment:
isn't it better to have it in place and not for the entire file ?

Reply:
yes